packages/www/hooks/use-api.tsx

```diff
@@ -313,6 +317,14 @@ const makeContext = (state: ApiState, setState) => {
       return [res, user as User | ApiError];
     },
 
+    // Get current Stripe product, allowing for development users that don't have any
+    getUserProduct(user: User) {
+      if (user.stripeProductId) {
+        return products[user.stripeProductId];
+      }
+      return Object.values(products)[0];
```

Comment:
I think we should not have this logic in production/if stripe is configured. Also, maybe it's better to access some specific product not to rely on object keys ordering. WDYT of this?
```suggestion
      if (hasStripe) {
        return products[user.stripeProductId];
      }
      return products[user.stripeProductId || "prod_0"];
```